resources/tagtypes/40.json

```diff
@@ -1,6 +1,6 @@
 {
 	"version": 0,
-	"name": "M3 2.9\" FREEZER",
+	"name": "M3 2.9\" BW",
```

Comment:
I'm happy to change this if there are other ideas. `M3 2.9\" BW` just seemed like the best compromise given that it will be used for both the 'freezer' and normal BW variant.